vllm/utils.py

```diff
@@ -543,11 +543,18 @@ def get_open_zmq_ipc_path() -> str:
     return f"ipc://{base_rpc_path}/{uuid4()}"
 
 
-def get_open_port() -> int:
+def get_open_port(force: bool = False) -> int:
     port = envs.VLLM_PORT
     if port is not None:
+        if force and port is not None:
```

Comment:
`port is not None` is already checked
```suggestion
        if force:
```
And you probably need another check about `if force and port is None`.

Reply:
Fixed in latest commit.